client/my-sites/post-selector/selector.jsx

```diff
@@ -185,20 +200,27 @@ const PostSelectorPosts = React.createClass( {
 		return includes( requestedPages, lastPage ) && ! loading;
 	},
 
-	getItemHeight( item ) {
-		if ( item && this.itemHeights[ item.global_ID ] ) {
-			return this.itemHeights[ item.global_ID ];
+	getPostChildren( postId ) {
+		const { posts } = this.props;
+		return filter( posts, ( { parent } ) => parent && parent.ID === postId );
+	},
+
+	getItemHeight( item, _recurse = false ) {
+		if ( ! item ) {
+			return ITEM_HEIGHT;
 		}
 
-		let height = ITEM_HEIGHT;
+		if ( item.parent && ! _recurse && includes( this.postIds, item.parent.ID ) ) {
+			return 0;
+		}
 
-		if ( item && item.items ) {
-			height += item.items.reduce( ( memo, nestedItem ) => {
-				return memo + this.getItemHeight( nestedItem );
-			}, 0 );
+		if ( this.itemHeights[ item.global_ID ] ) {
```

Comment:
Hmm, if I recall correctly, the idea with `setItemRef` is to [update the cached height](https://github.com/Automattic/wp-calypso/blob/f34aec2de1a5b96f6685e64ee8c6fc76ac848196/client/my-sites/post-selector/selector.jsx#L138-L141), even if it already existed, and to queue up a height recompute to the underlying `VirtualScroll` if anything has changed from current expectations. For this to work correctly, would require that `setItemRef` is called on later `renderItem` calls, which I should verify to be the case but I hadn't encountered any issues with it.